opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetrySpanBuilderImpl.java

```diff
@@ -164,7 +164,11 @@ public Span startSpan() {
       otelSpanBuilder.setParent(Context.current().with((OpenTelemetrySpanImpl) ocParent));
     }
     if (ocRemoteParentSpanContext != null) {
-      otelSpanBuilder.addLink(SpanConverter.mapSpanContext(ocRemoteParentSpanContext));
+      io.opentelemetry.api.trace.SpanContext spanContext =
+          SpanConverter.mapSpanContext(ocRemoteParentSpanContext, /* isRemoteParent= */ true);
+      otelSpanBuilder.setParent(
+          Context.current().with(io.opentelemetry.api.trace.Span.wrap(spanContext)));
+      otelSpanBuilder.addLink(spanContext);
```

Comment:
Why adding link also?

Reply:
Wasn't sure if anyone is dependent on the existing behavior.

Reply:
Do you think we'd remove the `addLink` before marking the opencensus shim stable?

Reply:
I'm not sure. I'd want the [spec opencensus compatibility doc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/opencensus.md) to say what to do here before marking stable. That doc is currently lax on details which we see in the [opentracing compatibility doc](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/opentracing.md), so I don't see this happening any time soon.